infrastructure/crashtests/testdriver.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html class="test-wait">
```

Comment:
The rfc says "testdriver in non-testharness tests will only be supported with `class=(ref)test-wait`". Do we need put this in the document in case that is not obvious?

Reply:
Done